code/game/mecha/mecha.dm

```diff
@@ -1175,12 +1171,11 @@ var/year_integer = text2num(year) // = 2013???
 /datum/action/innate/mecha/mech_toggle_thrusters/Activate()
 	if(!owner || !chassis || chassis.occupant != owner)
 		return
-	var/obj/mecha/M = chassis

```

Comment:
The var used to be necessary when only certain mech type had access to certain mech actions, it's now useless.